docs/cloud/dev/spryker-cloud-commerce-os/verifying-email-addresses.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Verifying email addresses
+description: Learn how to verify email addresses, so that your applications can send emails from them.
+template: howto-guide-template
+---
+
+For your applications to be able to send emails from an email address, you need to verify it.
```

Comment:
If the production mode is not active for your environment to prevent spam on non-production environments, in order for you application to.... etc